app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -161,8 +161,10 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.prompt.hidden ? this.prompt.hidden = false : this.prompt.hidden = true
 
     if (!this.activeForm.hidden) {
-      this.activeForm.querySelectorAll('.gem-c-textarea .govuk-textarea, .gem-c-input.govuk-input')[0]
-        .focus()
+      var input = this.activeForm.querySelectorAll('.gem-c-textarea .govuk-textarea, .gem-c-input.govuk-input')
+      if (input) {
```

Comment:
Ah, you'll need to check the length of the result, because `querySelectorAll(.whargleblargle)` returns `[]`, and `if([]) console.log('truthy')` logs "truthy".

```suggestion
      if (input.length > 0) {
```